debian/changelog

```diff
@@ -1,3 +1,20 @@
+ubuntu-core-initramfs (56) jammy; urgency=medium
+
+  [ Valentin David ]
+  * Update systemd to 249.11-0ubuntu3
+  * Fix recovery chooser trigger on NUC
+  * Print warnings when modules are in the wrong configuration file
```

Comment:
I think for the perspective of the user we should say something like "Do not force load kernel modules that can be automatically loaded".

Reply:
or "explicitly load"